static/app/views/issueDetails/traceTimeline/traceTimeline.spec.tsx

```diff
@@ -94,7 +108,6 @@ describe('TraceTimeline', () => {
     expect(useRouteAnalyticsParams).toHaveBeenCalledWith({
       has_related_trace_issue: false,
       trace_timeline_status: 'shown',
-      trace_timeline_two_issues: true,
```

Comment:
We're not going to report this metric anymore. It was to compare to the previous trace timeline CTR.